tests/conftest.py

```diff
@@ -45,34 +45,34 @@
 
 
 ltlf_formulas = [
-    "A",
-    "!A",
-    "A | B",
-    "A & B",
-    "X A",
-    "WX A",
-    "(X A) <-> !(WX(!A))",
-    "(WX A) <-> !(X(!A))",
-    "A U B",
-    "A R B",
-    "F A",
-    "F(A & B)",
-    "F(A | B)",
-    "F(A & last)",
-    "G A",
-    "F(G(A))",
-    "G(F(A))",
+    "a",
+    "!a",
+    "a | b",
+    "a & b",
+    "X a",
+    "WX a",
+    "(X a) <-> !(WX(!a))",
+    "(WX a) <-> !(X(!a))",
+    "a U b",
+    "a R b",
+    "F a",
+    "F(a & b)",
+    "F(a | b)",
+    # "F(a & last)",
+    "G a",
+    "F(G(a))",
+    "G(F(a))",
     "G(true)",
     "F(true)",
     "G(false)",
     "F(false)",
-    "last",
-    "end",
+    # "last",
+    # "end",
```

Comment:
reintroduce them?

Reply:
Yes, sure! Maybe we need a new LTLfLast class, and link to the existing LTLfEnd.

Reply:
We do have `LTLfEnd` https://github.com/whitemech/flloat/blob/develop/flloat/ltlf.py#L543
But not `LTLfLast` (which is `LTLfWeakNext(LTLfFalse())`.

Ok so maybe reintroduce only `end`? `last` is for another PR?

Reply:
I can do this before the merge.

Reply:
> Ok so maybe reintroduce only `end`? `last` is for another PR?

I should be able to address both in the same commit.